index.bs

```diff
@@ -102,7 +102,7 @@ the existence, of credentials scoped to other [RPS].
 
 [RPS] employ the [=Web Authentication API=] during two distinct, but related, [=ceremonies=] involving a user. The first
 is [=Registration=], where a [=public key credential=] is created on an [=authenticator=], and associated by a [=[RP]=]
-with the present user's account (the account may already exist or may be created at this time). The second is
+with the present user's account (the account MAY already exist or MAY be created at this time). The second is
```

Comment:
Please read through the changes you're making here and check that you actually intend the RFC 2119 meanings. This change is in a non-normative section, so it's definitely not right.